### PR TITLE
Fix badges in README.md

### PR DESCRIPTION
**Pull request by orioncohen**
## Description
Address problems with badge display, issue #3. Work in progress.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] display badges correctly
  - [x] add desired badges
  - [ ] integrate CI badges with git CI
  - [x] roughly verify codecov percentage

## Status
- [x] Ready to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 SolvationAnalysis
 ==============================
 [//]: # (Badges)
-[![GitHub Actions Build Status](https://github.com/REPLACE_WITH_OWNER_ACCOUNT/solvation_analysis/workflows/CI/badge.svg)](https://github.com/REPLACE_WITH_OWNER_ACCOUNT/solvation_analysis/actions?query=workflow%3ACI)
-[![codecov](https://codecov.io/gh/REPLACE_WITH_OWNER_ACCOUNT/SolvationAnalysis/branch/master/graph/badge.svg)](https://codecov.io/gh/REPLACE_WITH_OWNER_ACCOUNT/SolvationAnalysis/branch/master)
+[![GitHub Actions Build Status](https://github.com/orioncohen/solvation_analysis/workflows/CI/badge.svg)](https://github.com/orioncohen/solvation_analysis/actions?query=workflow%3ACI)
```
Comment (lilyminium) [inline README.md:4]: In this case, the owner account is MDAnalysis -- that may be why badges are not resolving :)

Comment (hmacdope) [reply README.md:4]: [Powered by NumFOCUS](https://www.numfocus.org/)
[Powered by MDAnalysis](https://www.mdanalysis.org)
[GitHub Actions Status](https://github.com/MDAnalysis/membrane-curvature/actions?query=workflow%3ACI)
[codecov](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main)

Is a few of them :)

Comment (lilyminium) [reply README.md:4]: @hmacdope I'm not sure that Orion can see the raw text behind the badges.

```
[![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://www.numfocus.org/)
[![Powered by MDAnalysis](https://img.shields.io/badge/powered%20by-MDAnalysis-orange.svg?logoWidth=16&logo=data:image/x-icon;base64,AAABAAEAEBAAAAEAIAAoBAAAFgAAACgAAAAQAAAAIAAAAAEAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJD+XwCY/fEAkf3uAJf97wGT/a+HfHaoiIWE7n9/f+6Hh4fvgICAjwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACT/yYAlP//AJ///wCg//8JjvOchXly1oaGhv+Ghob/j4+P/39/f3IAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJH8aQCY/8wAkv2kfY+elJ6al/yVlZX7iIiI8H9/f7h/f38UAAAAAAAAAAAAAAAAAAAAAAAAAAB/f38egYF/noqAebF8gYaagnx3oFpUUtZpaWr/WFhY8zo6OmT///8BAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgICAn46Ojv+Hh4b/jouJ/4iGhfcAAADnAAAA/wAAAP8AAADIAAAAAwCj/zIAnf2VAJD/PAAAAAAAAAAAAAAAAICAgNGHh4f/gICA/4SEhP+Xl5f/AwMD/wAAAP8AAAD/AAAA/wAAAB8Aov9/ALr//wCS/Z0AAAAAAAAAAAAAAACBgYGOjo6O/4mJif+Pj4//iYmJ/wAAAOAAAAD+AAAA/wAAAP8AAABhAP7+FgCi/38Axf4fAAAAAAAAAAAAAAAAiIiID4GBgYKCgoKogoB+fYSEgZhgYGDZXl5e/m9vb/9ISEjpEBAQxw8AAFQAAAAAAAAANQAAADcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAjo6Mb5iYmP+cnJz/jY2N95CQkO4pKSn/AAAA7gAAAP0AAAD7AAAAhgAAAAEAAAAAAAAAAACL/gsAkv2uAJX/QQAAAAB9fX3egoKC/4CAgP+NjY3/c3Nz+wAAAP8AAAD/AAAA/wAAAPUAAAAcAAAAAAAAAAAAnP4NAJL9rgCR/0YAAAAAfX19w4ODg/98fHz/i4uL/4qKivwAAAD/AAAA/wAAAP8AAAD1AAAAGwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAALGxsVyqqqr/mpqa/6mpqf9KSUn/AAAA5QAAAPkAAAD5AAAAhQAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADkUFBSuZ2dn/3V1df8uLi7bAAAATgBGfyQAAAA2AAAAMwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB0AAADoAAAA/wAAAP8AAAD/AAAAWgC3/2AAnv3eAJ/+dgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA9AAAA/wAAAP8AAAD/AAAA/wAKDzEAnP3WAKn//wCS/OgAf/8MAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIQAAANwAAADtAAAA7QAAAMAAABUMAJn9gwCe/e0Aj/2LAP//AQAAAAAAAAAA)](https://www.mdanalysis.org)
[![GitHub Actions Status](https://github.com/MDAnalysis/solvation-analysis/workflows/CI/badge.svg)](https://github.com/MDAnalysis/solvation-analysis/actions?query=workflow%3ACI)
[![codecov](https://codecov.io/gh/MDAnalysis/solvation-analysis//branch/main/graph/badge.svg)](https://codecov.io/gh/MDAnalysis/solvation-analysis//branch/main)

Is a few of them :)
```

Comment (hmacdope) [reply README.md:4]: Ah sorry didn't know that @lilyminium. Thanks for clarifying.

**Comment by hmacdope** [comment]
@orioncohen looks like you may have merged your two PR changes by accident? The diff seems to include changes in #6.

Don't sweat, I do this all the time. Personally, I would just hack this back to a clean start by copying some files around from an earlier commit state. Or alternately, you can revert some commits, this is a bit more liable to be confusing though. Dont worry too much about your git history in individual PRs as we will squash anyway.  :)

**Comment by orioncohen** [comment]
> @orioncohen looks like you may have merged your two PR changes by accident? The diff seems to include changes in #6.
> 
> Don't sweat, I do this all the time. Personally, I would just hack this back to a clean start by copying some files around from an earlier commit state. Or alternately, you can revert some commits, this is a bit more liable to be confusing though. Dont worry too much about your git history in individual PRs as we will squash anyway. :)

whoops! since there is so little code here I elected to just force push to overwrite the old commits

File: README.md (hunk 1 above)
Comment (orioncohen) [reply README.md]: Thanks for the templates! These should be up now. I will need to update the CI badges once I get CI set up. I am unsure how I can have 62% codecov when I haven't put up any unit testing yet?

Comment (hmacdope) [reply README.md]: I think as you don't have much code yet the import statement covers most if it.

**Review by hmacdope — APPROVED** [review]

**Comment by hmacdope** [comment]
I'm happy with this @orioncohen, feel free to **squash** and merge.

**Pull request merged**